CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+
+**If you want to join and support us**, just [write us at hello@boxwise.co](mailto:hello@boxwise.co) and include your github handle and / or CV!
+
+**Do you need help using Boxwise, or you found a bug, or got an idea for a feature?** [Please write us at helpme@boxwise.co!](mailto:helpme@boxwise.co)
+
+### How do we work in general?
+
+Quick summary:
+- We follow [Agile methods](https://medium.com/@svharivinod/rolling-out-features-for-zomato-using-agile-8f24bba1ddd8) ([short video](https://www.youtube.com/watch?v=QCif_-r8eK4)) to roll out new features. 
+- Our sprint cycle is about a month. In this month we meet twice online - once in the middle of the month to check-in and discuss problems, once at the end/beginning of the month to demo our progress of the sprint. We ask you to join at least the demo call once per month.
+- We use slack as our main communication tool and do asynchronous [stand-ups](https://en.wikipedia.org/wiki/Stand-up_meeting) there once a week.
+- On Fridays, we offer a space for troubleshooting if you are stuck and need to dicuss something about your task.
+
+### How do I get my tasks? Where do I find information about them?
+We are using trello for Task Management. Our [product manager](https://www.youtube.com/watch?v=502ILHjX9EE) assigns the tickets to everybody taking part in a sprint. Each ticket should include a user story to understand why we need the ticket. If you are missing information, do not feel shy to write in slack.
+
+### How do I start with my task?
+We would like to do [test driven development](https://www.browserstack.com/guide/tdd-vs-bdd-vs-atdd) where possible. Thinking about the tests first forces you to think about a clear image how a screen or endpoint must look like.
+Besides that, check our [guidelines](#guidelines) and [development principles](#development-principles).
+
+### I'm stuck. How do I get help?
+Basically write in the #dev channel in slack or come to the troubleshooting meeting on Friday mentioned above.
+Please write in **public** channels and tag people instead of writing them privately. It is completely ok to be stuck and ask for help and this way others with the same problem find the solution easier.
+
+If you want somebody to look at your code when you are stuck, then [**create a draft Pull Request**](https://github.blog/2019-02-14-introducing-draft-pull-requests/). It gives the others a quick insight in what you have done and is great to directly comment on the code.
+
+### I'm finished with my task. What now?
+If you haven't used GitHub before, [this is a tutorial](http://makeapullrequest.com/) that will help. Don't be afraid to ask for help, if you need it. (Everyone is a beginner at some point!)
+
+- Commit all your progress on a branch which you started from master. [Here, guidelines how to write good commit messages](https://github.com/erlang/otp/wiki/writing-good-commit-messages).
```

Comment:
We should say something about Trunk Based Development (https://trunkbaseddevelopment.com/), and keeping your local source in sync with master before you start working every day.

Reply:
added some comments.